lib/bitcrowd_ecto/changeset.ex

```diff
@@ -316,4 +331,83 @@ defmodule BitcrowdEcto.Changeset do
       changeset
     end
   end
+
+  @doc "Validates Money according to the given validator"
+  @doc since: "0.13.0"
+  @spec validate_money(Changeset.t(), atom(), validate_money_option) ::
+          Changeset.t() | no_return()
+  def validate_money(changeset, field, [{validator, target}]) do
+    ensure_money_loaded!()
+
+    validate_change(changeset, field, fn
+      _, %Money{} = value -> validate_money(field, value, validator, target)
+      _, _not_money -> raise "given field must be Money"
+    end)
+  end
+
+  defp validate_money(field, value, :equal_to, target) do
+    if Money.compare(value, target) != :eq do
+      [{field, {"must be equal to %{money}", [money: target, validation: :equal_to]}}]
+    else
+      []
+    end
+  end
+
+  defp validate_money(field, value, :less_than, target) do
+    if Money.compare(value, target) in [:eq, :gt] do
+      [{field, {"must be less than %{money}", [money: target, validation: :less_than]}}]
+    else
+      []
+    end
+  end
+
+  defp validate_money(field, value, :more_than, target) do
+    if Money.compare(value, target) in [:eq, :lt] do
+      [{field, {"must be more than %{money}", [money: target, validation: :more_than]}}]
+    else
+      []
+    end
+  end
+
+  defp validate_money(field, value, :less_than_or_equal_to, target) do
+    if Money.compare(value, target) == :gt do
+      [
+        {field,
+         {"must be less than or equal to %{money}",
+          [money: target, validation: :less_than_or_equal_to]}}
+      ]
+    else
+      []
+    end
+  end
+
+  defp validate_money(field, value, :more_than_or_equal_to, target) do
+    if Money.compare(value, target) == :lt do
+      [
+        {field,
+         {"must be more than or equal to %{money}",
+          [money: target, validation: :more_than_or_equal_to]}}
+      ]
+    else
+      []
+    end
+  end
+
+  defp validate_money(field, %Money{currency: currency}, :currency, target) do
+    if currency != target do
+      [{field, {"currency must be %{currency}", [currency: target, validation: :currency]}}]
+    else
+      []
+    end
+  end
+
+  defp validate_money(_field, _value, validator, _money) do
+    raise "Unknown money validator '#{validator}'. Existing validators are #{inspect(@money_validators)}."
+  end
+
+  defp ensure_money_loaded! do
+    unless Code.ensure_loaded?(Money) do
+      raise "The 'Money' library is require to use the validator"
+    end
+  end
```

Comment:
👍  Do you want to mention it on the readme that this is an optional dependency and how to install?